src/inputs/BasicInputs/BasicEnumInput/BasicEnumInput.js

```diff
@@ -28,7 +28,7 @@ export const BasicEnumInput = (props) => {
       {...otherProps}
     >
       <TextField
-        data-cy="text_field"
+        data-cy="text-field"
```

Comment:
This change may break existing tests using this id, is it really necessary ?

Reply:
+1

Reply:
In fact, this data-cy is used in only one place. I think we can modify it in the component and in the test id files without breaking any other tests.
Then, as we upgrade as a major version, we can make breaking changes ?